egs/public_dataset/kiritan/run.sh

```diff
@@ -6,11 +6,12 @@
 . ./cmd.sh || exit 1;
 
 
-stage=0
-stop_stage=100
+stage=3
+stop_stage=5
 ngpu=1
 raw_data_dir=downloads
 expdir=exp/rnn_norm_perp
+download_wavernn_vocoder=True
```

Comment:
do you have some modification to the original wavernn repo? If so, you may push your changes to here: https://github.com/SJTMusicTeam/WaveRNN